app/server/appsmith-plugins/dynamoPlugin/src/main/java/com/external/plugins/DynamoPlugin.java

```diff
@@ -337,10 +337,12 @@ else if (rawResponse.get(topLevelKey) instanceof Map) {
                 System.out.println(Thread.currentThread().getName() + ": In the DynamoPlugin, got action execution result");
                 return result;
             })
-                    .onErrorResume(AppsmithPluginException.class, error  -> {
+                    .onErrorResume(Exception.class, error  -> {
```

Comment:
I think we can omit the first argument to this method, when we want to handle any exception type.

```suggestion
                    .onErrorResume(error  -> {
```

Reply:
Fair :D I shall make the change across!